.github/workflows/ci.yml

```
@@ -11,7 +11,7 @@ jobs:
   specs:
     runs-on: ubuntu-latest
 
-    name: Ruby ${{ matrix.ruby-version }}, Rails ${{ matrix.ruby-version }}, LCH ${{ matrix.legacy_connection_handling }}
+    name: Ruby ${{ matrix.ruby-version }}, Rails ${{ matrix.gemfile }}, LCH ${{ matrix.legacy_connection_handling }}
```

Actually, since `matrix.gemfile` includes the word Rails already
```suggestion
    name: Ruby ${{ matrix.ruby-version }}, ${{ matrix.gemfile }}, LCH ${{ matrix.legacy_connection_handling }}
```